src/spotify.ts

```diff
@@ -0,0 +1,139 @@
+const completionSpec: Fig.Spec = {
+  name: "spotify",
+  description: "CLI to use Spotify from the terminal",
+  subcommands: [
+    {
+      name: "play",
+      description: "Resumes playback where Spotify last left off",
+      args: {
+        name: "song name",
+        description: "Finds a song by name and plays it",
```

Comment:
The arg description is meant to refer to the arg content, not to the action performed by the subcommand/option of which the argument is part:
```suggestion
        description: "The name of the song to start playing",
```